packages/accounts/src/kernel-zerodev/__tests__/kernel-account.test.ts

```diff
@@ -198,22 +196,31 @@ describe("Kernel Account Tests", () => {
   //     await expect(result).resolves.not.toThrowError();
   // });
 
+  // it("sendUserOperation batch should execute properly", async () => {
```

Comment:
@ManaOmlabs , the aa infra on alchemy is now open to the public so it should be possible to make these tests pass here. However, @avasisht23 is working on splitting tests into integration vs unit tests so we'll make sure to port these over as integration tests since they require an alchemy api key

Reply:
Sure thing. I will be on lookout for the respective PR

Reply:
Handled